rdr_service/resource/schemas/participant.py

```diff
@@ -320,7 +332,8 @@ class ParticipantSchema(Schema):
     races = fields.Nested(RaceSchema, many=True)
     genders = fields.Nested(GenderSchema, many=True)
     modules = fields.Nested(ModuleStatusSchema, many=True)
-    consents = fields.Nested(ConsentSchema, many=True)
+    # TODO: Deprecated, but leave around until BigQuery table support is removed.
+    # consents = fields.Nested(ConsentSchema, many=True)
```

Comment:
Per comments above, are we ready to remove this yet or do we need to give users time to update the queries?

Reply:
I want to leave it in for a short while, just in case we need to revert for some reason.